acceptance/tests/environment/environment_scenario-non_existent.rb

```diff
@@ -46,9 +46,9 @@
   },
   :puppet_agent => {
     :exit_code => 1,
-    :matches => [%r{Warning.*404.*Could not find environment '#{env}'},
+    :matches => [%r{(Warning|Error).*(404|400).*Could not find environment '#{env}'},
```

Comment:
yeah, i was surprised too. maybe it's a "bad request" because we are asking for an environment that doesn't exist?